pyccl/power.py

```diff
@@ -108,3 +108,17 @@ def sigma8(cosmo):
     """
     cosmo.compute_linear_power()
     return sigmaR(cosmo, 8.0 / cosmo['h'])
+
+
+def kNL(cosmo, a):
+    """Scale for the non-linear cut.
+
+    Args:
+        cosmo (:class:`~pyccl.core.Cosmology`): Cosmological parameters.
+        a (float or array_like): Scale factor(s), normalized to 1 today.
+
+    Returns:
+        float or array-like: Scale of non-linear cut-off; Mpc^-1.
```

Comment:
```suggestion
        float or array-like: Scale of non-linear cut-off; Mpc^-1.
    Note:
        k_NL is calculated based on Lagrangian perturbation theory as the 
        inverse of variance of the displacement field, i.e 
           k_NL = 1/Sigma = [1/(6 pi^2)  int P_L(k) dk]^{-1/2}     
```

Reply:
Just to check -- is this "Note:" type tag compatible with ReadTheDocs, as I can't see it being used anywhere else in the CCL/pyccl directory? What I see intead is ".. note::" being used. Can you advise @slosar ?

Reply:
I've implemented the above with the ".. note::" syntax.